composer/core/state.py

```diff
@@ -144,7 +145,7 @@ def __init__(
         self.model = model
         self.grad_accum = grad_accum
         self.train_dataloader = train_dataloader
-        self.eval_dataloader = eval_dataloader
+        self.evaluators = evaluators
```

Comment:
`self.eval_dataloader`, _not_ `self.evaluator` here

Reply:
As discussed offline - we're going to leave it as self.evaluators and change it in a further PR